style.css

```diff
@@ -121,3 +121,16 @@ body > .is-root-container,
 	margin-right: auto !important;
 	width: inherit;
 }
+
+/*
+ * Responsive menu container padding.
+ * This gives the responsive container the same padding as the site itself.
```

Comment:
@jasmussen, do you think this will ever be something that happens by default? If so I can add a code comment here and point to an issue.

Reply:
For a modal overlay that covers the screen, yes I think it makes sense for it to inherit a global padding variable, OR at least have that padding variable easily configured via theme.json (and either would be good for the purposes of block themes, right?)

I can't point to a ticket you could link here, though! I think there are some pieces to work out around how separate overlay menu editing will happen. My current mental model is that it can function a bit like how editing a template part in isolation does, and this feature will likely have to land before the theme.json integration.

By the way, https://github.com/WordPress/gutenberg/pull/37323 landed which should simplify some things for this PR (not that I think you need ot do anything different), and I applied the backport label for TT2's usage.

Reply:
Ok, thanks. I clarified the code comment a little bit, but I'm not sure if I should be more explicit: 

```
This ensures the responsive container inherits the same
spacing defined above. This behavior may be built into
the Block Editor in the future.
 ```

Reply:
Seems good! It serves also partially as a reminder to our future selves. 👍 👍